forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/account/v1_1/standingorders/StandingOrdersApiController.java

```diff
@@ -91,7 +88,6 @@
                 PageRequest.of(page, PAGE_LIMIT_STANDING_ORDERS));
         List<OBStandingOrder1> standingOrders = standingOrdersResponse.stream()
                 .map(so -> toOBStandingOrder1(so.getStandingOrder()))
-                .map(so -> accountDataInternalIdFilter.apply(so))
```

Comment:
I've removed a small amount of ID filtering from v1.0 and v2.0 because the objects do not contain a creditorAccount, meaning I cannot differentiate between domestic and international. However, this does not impact any of the functionality from v3.0 onwards.